src/components/custom/BalanceChange/BalanceChange.tsx

```diff
@@ -0,0 +1,38 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import { Typography } from '@/components/custom/Typography';
+
+interface BalanceChangeProps {
+  delta: number;
+  percentChange: string;
+  timePeriod: string;
+  className?: string;
+}
+
+export const BalanceChange: React.FC<BalanceChangeProps> = ({
+  delta,
+  percentChange,
+  timePeriod,
+  className,
+}) => {
+  const sign = delta >= 0 ? '+' : '-';
+  const formattedDifference = `${sign} ${Math.abs(delta).toLocaleString()}`;
+
+  return (
+    <span
+      className={classNames(
+        'flex flex-row mt-2',
+        className,
+        'DollarValueChange-root'
```

Comment:
make sure to update the classname

Reply:
what about something like yesterday. that dont work no more

Reply:
? the suggested change here is
```suggestion
        'BalanceChange-root'
```